tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -355,8 +357,23 @@ export class WebGPUBackend extends KernelBackend {
     return this.currentComputePass;
   }
 
+  // Check if parallel compilation is done.
+  async checkCompileCompletion() {
```

Comment:
Should we rename it to `checkCompileCompletionAsync` to keep consistent with webgl https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgl/src/backend_webgl.ts#L1228?